examples/mesos-client-example/src/test/scala/com/mesosphere/mesos/examples/MesosClientExampleFrameworkTest.scala

```diff
@@ -9,7 +9,6 @@ import org.junit.runner.RunWith
 import org.scalatest.junit.JUnitRunner
 import java.util
 
-@RunWith(classOf[JUnitRunner])
```

Comment:
This means the imports for `RunWith` and `JUnitRunner` are unused and hence can be removed. I thought our scalaconf catches these (or does it ignore cuz these are test files ?!)

Reply:
🤔

Reply:
btw. the build failed so everything is awesome. It was not failing only because the build was stopped on core tests failing so it did not get to this point (at least that's my understanding)